modules/apps/frontend-js/frontend-js-react-web/types/src/main/resources/META-INF/resources/js/test.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Can we remove this file?

Reply:
I don't even understand what this file is, it says it had the test for `getCountries` and I've apparently deleted that in a commit, but 

<img src="https://media0.giphy.com/media/aZ3LDBs1ExsE8/giphy.gif"/>